.changeset/many-lemons-arrive.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core-components': patch
+---
+
+Fixed bug in Select component where a broken filter was showing when an invalid initialState was passed.
```

Comment:
spelling issue to fix - you can also highlight that Select is the component name by formating it as code. suggestion would be:

```suggestion
Fixed `Select` component default filter state.
```

Reply:
Tjoo, thanks for the warm welcome and your feedback. 🤩
I'll give it a shot, new PR coming tomorrow morning.